client/helpers_test.go

```diff
@@ -177,3 +177,36 @@ func TestWaitForJobTimeout(t *testing.T) {
 		t.Fatal("Error was nil!")
 	}
 }
+
+func TestContains(t *testing.T) {
```

Comment:
good test! - but- for future reference, we probably don't need unit tests for helpers that are this simple

Reply:
Yea I thought it was overkill writing a test for a basically a for loop too. Making up for not writing tests all those years 😅